src/Query/Providers/QueryProvider.cs

```diff
@@ -344,7 +344,23 @@ public int ExecuteNonQuery(string query, CommandParameterCollection parameters)
         }
 
         /// <inheritdoc/>
-        public async Task<int> ExecuteNonQueryAsync(string query) => await ExecuteNonQueryAsync(query, null);
+        public async Task<int> ExecuteNonQueryAsync(string query)
+            => await ExecuteNonQueryAsync(query, new CommandParameterCollection());
```

Comment:
No need to create an instance.

```suggestion
            => await ExecuteNonQueryAsync(query, (CommandParameterCollection)null);
```